test/passing/refs.ahrefs/break_fun_decl-smart.ml.ref

```diff
@@ -129,8 +129,12 @@ let f
 
 let new_specialised_args =
   Variable.Map.mapi
-    (fun new_inner_var______ (definition : Definition.t) :
-         Flambda.specialised_to -> ())
+    (fun
+      new_inner_var______
+      (definition : Definition.t)
+      :
+      Flambda.specialised_to
+    -> ())
```

Comment:
This is not great but it is consistent. I would like to box the arguments, so that 
```
fun
  arg1 arg2
  : return_type
->
  body
```
is allowed.